lib/src/components/NxButton/NxButton.tsx

```diff
@@ -17,7 +17,13 @@ const NxButton = forwardRef<HTMLButtonElement, Props>(
         'nx-btn--inline': inline
       });
 
-      return <button ref={ref} className={classNames} {...attrs}>{children}</button>;
+      return (
+        <button aria-disabled={classNames.includes('disabled')}
```

Comment:
We still have to sort-of support IE11 for a few more weeks.  Using `.includes` (a method that IE11 doesn't have) will break any page that uses an NxButton.  You'll need to use a different method.  I recommend ramda's [includes](https://ramdajs.com/docs/#includes) since ramda is our main utility library.

Reply:
aa854ba8d2db35c24c4843076f18f136cde59aa8